lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -138,6 +162,20 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (event.eventBridge.retryPolicy) {
+              RetryPolicy = _.mapKeys(
+                RetryPolicy,
+                (value, key) => key[0].toLocaleUpperCase() + key.slice(1)
+              );
+            }
+
+            if (event.eventBridge.retryPolicy) {
```

Comment:
To be similar to the rest of the code here, you should do this, no?

```suggestion
            if (DeadLetterConfig) {
```

BTW, I think there was some copy and paste mistake. This `if` is about the DLQ, not the retry policy.

Reply:
I'll push a commit to fix this. 🤓

Reply:
I tried to push here, but have no permission. Ended up pushing to https://github.com/OmnipresentGroup/serverless/pull/1 instead. Hope you don't mind. :)